doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -160,6 +165,52 @@ To use the sensor trigger, complete the following steps:
 .. note::
     |only_configured_module_note|
 
+Enabling passive power management
+=================================
+
+The |sensor_manager| can react to ``power_down_event`` and ``wake_up_event``.
+This functionality is called passive power management.
+
+To use the passive power management in |sensor_manager|, complete the following steps:
+
+1. Enable :kconfig:`CONFIG_CAF_SENSOR_MANAGER_PM` option.
+   If a sensor of your choice supports trigger functionality,its enabled and configured by you, there is nothing more to do.
+   If you did not enable trigger functionality for your sensor, continue with next steps.
```

Comment:
`support` and `enable` mean two different things, but here it looks like they're used as synonyms?